ide/main/src/content/remote-selenium-commandhandlers.js

```diff
@@ -373,7 +373,7 @@ function AccessorResult(result) {
   } else if (Deferred.isPromise(result)) {
     //We have a deferred, we need to change the terminating condition to include the resolution of the deferred
     result.done(function(arg) {
-      self.result = arg;
+      this.result = arg;
```

Comment:
this does not fix the problem, you would need to define above this function var self =this; and revert this change. or change the line below to `}.bind(this));`